### ISSUE TITLE
signature verification failed

### ISSUE THREAD

**Issue by hardil61** [micahflee/torbrowser-launcher#336]
try to install from Cerebrus linux Desktop menu the following error appears

Signature Verification Failed
Error code:Generic_verify_fail

**Comment by rogers0** [comment]
I meet the same error.
The key is expired?

```
$ gpg --verify tor-browser-linux64-7.5.6_en-US.tar.xz.asc
gpg: assuming signed data in 'tor-browser-linux64-7.5.6_en-US.tar.xz'
gpg: Signature made Sun 24 Jun 2018 04:36:16 AM JST
gpg:                using RSA key 0xD1483FA6C3C07136
gpg: Good signature from "Tor Browser Developers (signing key) <torbrowser@torproject.org>" [unknown]
gpg: Note: This key has expired!
Primary key fingerprint: EF6E 286D DA85 EA2A 4BA7  DE68 4E2C 6E87 9329 8290
     Subkey fingerprint: A430 0A6B C93C 0877 A445  1486 D148 3FA6 C3C0 7136
```

**Comment by micahflee** [comment]
This is fixed in the new version 0.3.0 that I just released. You can get it by either [installing from source](https://github.com/micahflee/torbrowser-launcher/blob/develop/BUILD.md) or installing from the Ubuntu PPA, described [here](https://github.com/micahflee/torbrowser-launcher/blob/develop/README.md#installing-in-ubuntu).
